apps/ios/Shared/Views/Chat/ComposeMessage/ComposeLinkView.swift

```diff
@@ -55,7 +55,7 @@ struct ComposeLinkView: View {
                     .foregroundColor(theme.colors.secondary)
             }
             .padding(.vertical, 5)
-            .frame(maxWidth: .infinity, minHeight: 60, maxHeight: 60)
```

Comment:
Setting height limit will make the view's text go out of bounds with larger sizes.

![Screenshot 2024-08-14 at 10 29 48](https://github.com/user-attachments/assets/e0f9449f-d174-4063-9441-17e0dfd5f1fc)